src/coloris.js

```diff
@@ -676,7 +684,6 @@
     // Render the UI
     picker = document.createElement('div');
     picker.setAttribute('id', 'clr-picker');
-    picker.className = 'clr-picker';
```

Comment:
This MUST NOT be removed. It will cause the picker to fail to display when the `theme` option is not set.

Reply:
Sorry, my bad. I only tried with options set!